staff/tachi-fernandez/cv/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curiiculum Vitae</title>
+</head>
+<body bgcolor= "white">
+  
+    <h1 style = color:blue> Curriculum Vitae => Tachi Fernandez</h1>
+      <h2>ºDatos personales:</h2>
+        <p>
+           <u>-Nombre completo : Tachi Fernandez Luna</u>
+             <br></br>
+           <u>-Direccion : mi casa nº8</u>
+             <br></br>
+           <u>-D.N.I. : 12345678R</u>
+             <br></br>
+           <u>-Nacionalidad : Puerto Rico</u>   
+        </p>
```

Comment:
- You can wrap from line 12 to line 21 inside a `<section>` tag
- Also, inside that `<section>`you can use a header with your `<h2>`